### PR TITLE
Feat delete collection#43

### LINKED ISSUE #43: feat: delete collection

### PR DESCRIPTION
**Pull request by svmi3195**
Since we use indices for paths I didn't remove collection entirely form the store, but replaced it with null.

closes #43

### CODE REVIEW

**Review by tomski80 — COMMENTED** [review]
this works, but there is no refresh after you delete collection (it all looks like nothing happen until you go to different page), and to user looks like it doesn't work. I think you could route to home after delete, you can use router.push('/')

**Comment by AntonTymczenko** [comment]
Please don't use path in router links. Use ` Name: ""`

**Comment by tomski80** [comment]
Shouldn't we add name to root path then ? There is no name for it in router.

**Comment by AntonTymczenko** [comment]
It is 'home'

**Comment by tomski80** [comment]
I've added router.push({ name: 'home' }) to deleteCollection method, so now it redirects home when collection is deleted.

**Comment by svmi3195** [comment]
Yeah, I had that thought about redirecting too, it makes sense.

File: src/store/index.js [hunk 1]
```diff
@@ -71,6 +74,11 @@ export default new Vuex.Store({
             Object.assign(state, JSON.parse(localStorage.getItem('store')))
           )
         }
-     }
+      }
+    },
+    actions: {
+        deleteCollection(context, id){
```
Comment (AntonTymczenko) [inline src/store/index.js:80]: you can use ES6 argument destructuring 
```
deleteCollection({ commit }, id) {
  commit('deleteCollection', id)
}
```
it is not so helpful in this case, but sticking to this technique would be helpful in more complicated instances

**Review by AntonTymczenko — APPROVED** [review]

**Pull request merged**